### DISCUSSION TITLE
Εβδομαδιαίο βίντεο κουϊζ

### DISCUSSION THREAD

**Discussion by epidrome** [courses-ionio/hci#1531] [Announcements]
Το 1ο εβδομαδιαίο βίντεο κουϊζ έχει ήδη αναρτηθεί με προθεσμία συμμετοχής και απάντησης μέχρι την Κυριακή 17 Οκτ.

Για να απαντήσετε θα πρέπει να κάνετε [σύνδεση με το σύστημα βίντεο κουϊζ](https://edpuzzle.com/join/cupsodl)

**Η εγγραφή σας στο σύστημα θα γίνει με το ionio.gr email και για επώνυμο μόνο τον ΑΜ σας, έτσι ώστε να μπορέσουμε να κάνουμε σωστά την καταγραφή της συμμετοχής και του βαθμού σας**

Η εργασία αυτή είναι εβδομαδιαία, επομένως θα πρέπει κάθε εβδομάδα να κάνετε σύνδεση στο παραπάνω σύστημα με προθεσμία την Κυριακή

Υπάρχουν διαθέσιμα δώδεκα βίντεο και αρκεί να απαντήσετε σε δέκα από αυτά

**Comment by CptMike** [comment]
Καλησπέρα κ.Χωριανόππουλε. Γράφετε πως έχουμε προθεσμία μεχρι τις 17 Οκτ., αλλά στο EdPuzzle γράφει μέχρι τις 21. Μήπως εννοούσατε μέχρι τις 21;

**Reply by epidrome** [reply]
επειδή αυτό το 1ο αργήσαμε να το ανακοινώσουμε δώσαμε παράταση μερικές μέρες έτσι ώστε όλοι να έχουν μια εβδομάδα για να το απαντήσουν

καλύτερα να μας γίνει συνήθεια η Κυριακή, που θα είναι πάντα η προθεσμία για τα επόμενα!

**Comment by p16foti** [comment]
Καλησπέρα κ.Χωριανόπουλε, συγκεκριμένα για το βίντεο κουιζ(edpuzzle), γράφετε ότι η εγγραφή μας θα γίνει με το ionio.gr και για επίθετο τον ΑΜ μας. Στο επίθετο έβαλα όπως ζητήσατε τον ΑΜ μου αλλά δεν μου ζήτησε το email(@ionio.gr) μονάχα username όπου προσπάθησα να βάλω για username το email μου(@ionio.gr) αλλά δεν με άφηνε οπότε έβαλα για username μόνο το p16foti και στη συνέχεια έχω συνδεθεί κανονικά αλλά χωρίς να έχει χρειαστεί να βάλω κάπου το email μου διότι δεν μου ζητήθηκε. Χρειάζεται να κάνω κάτι άλλο? ή τα στοιχεία που έβαλα(εκτός του email @ionio.gr) είναι αρκετά ώστε να γίνει καταγραφή της συμμετοχής μου?

**Reply by Sola1r3** [reply]
https://github.com/courses-ionio/hci/discussions/1533#discussioncomment-1476027

Έχει απαντήσει εδώ, ελπίζω να σε βοήθησα.

**Comment by P18kara8** [comment]
Εχω δει το βίντεο και έχω απαντήσει στις ερωτήσεις. Στην αναφορα που θα κάνω μέσα στο Branch μου τι πρέπει να βάλω? Βάζω κάποιο link της σελίδας ή τα στοιχεία μου στην πλατφόρμα της Edpuzzle?

**Reply by CptMike** [reply]
Νομίζω τα βειντεο κουζ δε μπαίνουν στην αναφορα.

**Comment by AlexandrosV20** [comment]
Έχω μια ερώτηση. Επειδή έβλεπα καταληκτική ημερομηνία μια μερα μετα την κυριακή και 2 βιντεάκια μου τα έκανα Δευτερες αυτά θα βαθμολογηθούν ή επειδή βγήκαν εκτός προθεσμίας για λίγες ώρες θα ακυρωθούν ; Επίσης θα ήθελα να ρωτήσω όταν βγεί ενα assignment εκτός προθεσμίας δεν κλειδώνεται αυτόματα όπως τα chapter στις ερωτήσεις κειμένου ;

**Reply by epidrome** [reply]
1η φορά χρησιμοποιούμε αυτό το σύστημα, φαίνεται πως οι προθεσμίες του είναι ενδεικτικές, αλλά ποιο το νόημα, αν κάποιος τα δει όλα μαζί του χρόνου; πότε θα προλάβει να κάνει συζήτηση και να σκεφτεί ή να εμπνευστεί και να ενημερώσει τα παραδοτέα του; 

όσα περισσότερα κάνετε τόσο καλύτερα! το μεγαλύτερο πρόβλημα που βλέπουμε αυτήν την στιγμή είναι πως (όπως πάντα, είτε τις διαλέξεις τις κάνω εγώ, είτε ο άλαν κέη!!!) από τους 110 φοιτητές την 1η εβδομάδα, τώρα είμαστε στους 25!

**Reply by AlexandrosV20** [reply]
Δεν τα κάνω όλα μαζί ένα ένα τα συμπληρώνω απλα σε 2 βιντεάκια εχει κοκκινήσει η ημερομηνία γιατί δεν είχα δει την ώρα 12:59 am καθώς στην λίστα με τα βιντεάκια δείχνει μόνο ημερομηνία και όχι ώρα. Θα σας επισυνάψω και σχετική φωτογραφία για να καταλάβετε τι εννοώ. Αν αυτά τα βιντεάκια που έιναι κόκκινα λόγω του ότι τα συμπλήρωσα την ίδια μέρα μετά τις 12:59 am αν μετράνε σαν completed ή οχι.
![image](https://user-images.githubusercontent.com/17967496/144867019-c5259bc5-fae3-4bb4-b7fe-98b9159130a9.png)

**Comment by p20exar** [comment]
Το ίδιο έπαθα κι εγώ. 'Εχουμε συνηθίσει στις περισσότερες εργασίες μας, η προθεσμία να λήγει βράδυ, στις 23:59 και δεν προσέχουμε το 12:59 am που σημαίνει μεσημέρι.
Από τα 12 συνολικά βιντεάκια μπορούμε να ασχοληθούμε με τα 10. Άρα είμαστε οκ. 
Το να χάσουμε 2 είναι μέσα στο λογικά πλαίσια νομίζω.  Πιστεύω πως τα κόκκινα δεν θα μετρήσουν, αλλά οκ, συνεχίζουμε..

(Προσοχή! εδώ γράφει ημέρα και ώρα λήξης. )

![image](https://user-images.githubusercontent.com/92602820/144912551-550df481-109e-487b-bd48-f14282f4a2e5.png)

**Reply by AlexandrosV20** [reply]
Το 12:59 am ειναι μια η ώρα το βραδυ για 'υτο και κοκκίνισαν τα βιντεακια, δηλαδή τέλος της Κυριακής ξημέρωμα Δευτέρας. Κύριε @epidrome επειδή με το thumb up δεν κατάλαβα στο προηγούμενο σχόλιο μου που κάνατε react, τελικά θα μετρήσουν αυτά που έχουν βγεί εκτός πριθεσμίας για μερικές ώρες ;

**Reply by epidrome** [reply]
ναι όσα είναι κοντά (π.χ., μερικές ώρες) στην προθεσμία μετράνε κανονικά

**Comment by AlexandrosV20** [comment]
Στο actively learn δεν αφήνει να κάνουμε αλλαγές στα στοιχεία πάντως.
